_plugins/ref_tag.rb

```diff
@@ -1,6 +1,5 @@
 # frozen_string_literal: true
```

Comment:
Maybe these three classes can be refactored into inheriting base functionality from a single one. :thinking:

Reply:
Will be addressed by #45 :+1: